pkg/blobplugin/Dockerfile

```diff
@@ -34,7 +34,9 @@ ARG ARCH=amd64
 RUN if [ "$ARCH" = "amd64" ] ; then \
   clean-install libcurl4-gnutls-dev && \
   wget -O /blobfuse-proxy/packages-microsoft-prod.deb https://packages.microsoft.com/config/ubuntu/18.04/packages-microsoft-prod.deb && \
-  dpkg -i /blobfuse-proxy/packages-microsoft-prod.deb && apt update && apt install blobfuse fuse -y && apt remove wget -y; fi
+  dpkg -i /blobfuse-proxy/packages-microsoft-prod.deb && apt update && apt install blobfuse blobfuse2 fuse -y && apt remove wget -y; fi
```

Comment:
blobfuse blobfuse2 fuse "fuse3" as well

Reply:
does `fuse3` package exist on ubuntu 18.04? I could not find it.
```
# apt update
Hit:1 http://azure.archive.ubuntu.com/ubuntu bionic InRelease
Get:2 http://azure.archive.ubuntu.com/ubuntu bionic-updates InRelease [88.7 kB]
Get:3 http://azure.archive.ubuntu.com/ubuntu bionic-backports InRelease [74.6 kB]
Hit:4 https://packages.microsoft.com/repos/azure-cli bionic InRelease
Hit:5 https://packages.microsoft.com/ubuntu/18.04/prod bionic InRelease
Get:6 https://aquasecurity.github.io/trivy-repo/deb bionic InRelease [3,054 B]
Hit:7 https://cli.github.com/packages bionic InRelease
Hit:8 https://download.docker.com/linux/ubuntu focal InRelease
Get:9 http://security.ubuntu.com/ubuntu bionic-security InRelease [88.7 kB]
Get:10 https://baltocdn.com/helm/stable/debian all InRelease [7,652 B]
Get:11 https://baltocdn.com/helm/stable/debian all/main amd64 Packages [2,872 B]
Fetched 266 kB in 1s (285 kB/s)
Reading package lists... Done
Building dependency tree
Reading state information... Done
89 packages can be upgraded. Run 'apt list --upgradable' to see them.
# apt search fuse3
Sorting... Done
Full Text Search... Done
# apt install fuse3
Reading package lists... Done
Building dependency tree
Reading state information... Done
E: Unable to locate package fuse3
# uname -a
Linux andydev 5.4.0-1077-azure #80~18.04.1-Ubuntu SMP Wed Apr 13 02:07:09 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
```

Reply:
No for Ubn-18 fuse3 is not available, but for Ubn20/22 it is. When you take blobfuse2 binary from ubn18 path it will be fuse2 compatible. just make sure we dont mix and match like taking blobfuse2 binary from ubn20 and packaging it for ubn18.

Reply:
thanks for the clarification, that means Ubuntu OS takes those dependencies, CSI driver won't need to care about the dependencies. We will install fues3 when Ubuntu 22.04 is supported on AKS.